### PR TITLE
Various XCache admonitions (SOFTWARE-5961)

### PR DESCRIPTION
**Pull request by matyasselmeci**

### CODE REVIEW

File: docs/data/stashcache/install-cache.md [hunk 1]
```diff
@@ -4,6 +4,10 @@ DateReviewed: 2023-03-03
 Installing the OSDF Cache
 =========================
 
+!!! warning "Deprecation warning"
+    This document is outdated and describes an XCache-based OSDF Cache install, which is deprecated.
+    Future OSDF Caches should be based on Pelican; documentation for a Pelican-based OSDF Cache install is forthcoming.
```
Comment (djw8605) [inline docs/data/stashcache/install-cache.md:9]: XCache based caches are deprecated, pelican is the future, but we have no docs on installing pelican caches...

Comment (matyasselmeci) [reply docs/data/stashcache/install-cache.md:9]: Yeah, we're working on it. In the meantime, though, we'd rather not have people setting up new XCaches...

**Review by brianhlin — APPROVED** [review]
LGTM!

**Comment by brianhlin** [comment]
CI failures unrelated, closing

**Pull request merged**